lib/radar_client.js

```diff
@@ -5,7 +5,9 @@ var MicroEE = require('microee'),
     StateMachine = require('./state.js'),
     immediate = typeof setImmediate != 'undefined' ? setImmediate :
                                     function(fn) { setTimeout(fn, 1); },
-    getClientVersion = require('./client_version.js');
+    getClientVersion = require('./client_version.js'),
+    Request = require('./message_request.js'),
```

Comment:
I would suggest the end goal here is to move them to a separate package so the api client can reuse it. But for now, let's keep this in this repo.